source/api/presentation/3.0/index.md

```diff
@@ -240,6 +240,25 @@ The value _MUST_ be a JSON array, with each item in the array being a JSON objec
 {"thumbnail": [{"id": "https://example.org/img/thumb.jpg", "type": "Image"}]}
 ```
 
+##### posterCanvas
+One or more Canvases providing content associated with the object represented by this resource but not part of that object. Examples include an image to show while a duration-only Canvas is playing audio; images, text and sound standing in for video content before the user initiates playback; or a film poster to attract user attention. The content provided by `posterCanvas` differs from a thumbnail: a client might use `thumbnail` to summarise and navigate multiple resources, then show content from `posterCanvas` as part of the presentation of a single resource. A poster Canvas can have different dimensions to the Canvas(es) of this resource.
+
+Clients _MAY_ display the content of a linked poster Canvas when presenting the resource, and if more than one is available _MAY_ choose the one most suited to the client user interface. When more than one Canvas is available, for example if `posterCanvas` is provided for the currently selected Range and the current Manifest, and the client is able to show a poster Canvas, the client _SHOULD_ pick the Canvas most specific to the content. Publishers _SHOULD NOT_ assume that `posterCanvas` content will be seen in all clients. Clients _SHOULD_ take care to avoid conflicts between time-based media in the `posterCanvas` and the content of the resource it is associated with.
+
+  * A Collection _MAY_ have one or more `posterCanvas` properties.<br/>
+   Clients _MAY_ render `posterCanvas` on a Collection.
+  * A Manifest _MAY_ have one or more `posterCanvas` properties.<br/>
+   Clients _MAY_ render `posterCanvas` on a Manifest.
+  * A Sequence _MAY_ have one or more `posterCanvas` properties.<br/>
+   Clients _MAY_ render `posterCanvas` on a Sequence.
+  * A Canvas _MAY_ have one or more `posterCanvas` properties.<br/>
+   Clients _MAY_ render `posterCanvas` on a Canvas.
+  * A Range _MAY_ have one or more `posterCanvas` properties.<br/>
+   Clients _MAY_ render `posterCanvas` on a Range.
+  * Other resource types _MUST NOT_ have a `posterCanvas`.<br/>
+   Clients _SHOULD_ ignore `posterCanvas` on other resource types.
```

Comment:
> Clients _SHOULD_ ignore `posterCanvas` on other resource types.

Why not _MUST_?

Reply:
Or leave out the last sentence altogether?

Reply:
I think we had this issue on something else in the last few days. I prefer not to have a _MUST_ for a case that isn't allowed anyway. I'd either keep as _SHOULD_ or else follow @tomcrane's suggestion and simple remove that sentence -- slight preference to remove

Reply:
The pattern at the moment is if the resource type MUST NOT have an `x`, then clients SHOULD ignore `x` on those types.

So I'm :+1: on the current text, or propose making a new issue if we want to change it across the board in section 3.

Reply:
Agreed on call that we'll leave as _SHOULD_